File: furniture-brownser/src/interface/Message.ts
export interface MessageInfo{
  name:string,
  telephone:string,
  email:string,
  text:string,
  itemList:number[]
}
export interface MessageResponse{
  id:number,
  username:string,
  name:string,
  telephone:string,
  email:string,
  text:string,
  createTime:string,
  itemList:string[]
}
export interface ItemResponse{
  id:number,
  name:string,
  status?:boolean
}

File: furniture-server/src/main/java/com/pvzer/furniture/pojo/Message.java
package com.pvzer.furniture.pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Message {
    private Integer id;
    private Integer userId;
    private String name;
    private String telephone;
    private String email;
    private String text;
    private LocalDateTime createTime;
    private List<Integer> itemList;

}
